x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/EsqlSpecTestCase.java

```diff
@@ -211,38 +202,6 @@ protected static void checkCapabilities(RestClient client, TestFeatureService te
         }
     }
 
-    protected static boolean hasCapabilities(List<String> requiredCapabilities) throws IOException {
-        return hasCapabilities(adminClient(), requiredCapabilities);
-    }
-
-    public static boolean hasCapabilities(RestClient client, List<String> requiredCapabilities) throws IOException {
-        if (requiredCapabilities.isEmpty()) {
-            return true;
-        }
-        try {
-            if (clusterHasCapability(client, "POST", "/_query", List.of(), requiredCapabilities).orElse(false)) {
-                return true;
-            }
-            LOGGER.info("capabilities API returned false, we might be in a mixed version cluster so falling back to cluster features");
-        } catch (ResponseException e) {
-            if (e.getResponse().getStatusLine().getStatusCode() / 100 == 4) {
-                /*
-                 * The node we're testing against is too old for the capabilities
-                 * API which means it has to be pretty old. Very old capabilities
-                 * are ALSO present in the features API, so we can check them instead.
-                 *
-                 * It's kind of weird that we check for *any* 400, but that's required
-                 * because old versions of Elasticsearch return 400, not the expected
-                 * 404.
-                 */
-                LOGGER.info("capabilities API failed, falling back to cluster features");
-            } else {
-                throw e;
-            }
-        }
-        return false;
-    }
```

Comment:
Moved.

Also, we don't need this big comment. It was require when this test supported versions of elasticsearch without the capability api.